### ISSUE TITLE
Q-time ignores the first (screen) touch after creation

### ISSUE THREAD

**Issue by alexeigs** [quasarframework/quasar#5624]
Issue: `q-time` ignores first click after mounting, if screen hasn't received any other touch since.

It's been hard to narrow it down and I'm obviously uncertain where the real issue is but the following hints I can share:

- Only happens in PWA mode when installed on the home screen on iOS; when run in the browser directlcy, the issue does not occur
- Does not occur on desktop/mouse clicks
- Loading a new page or dialog with a `q-time` component makes `q-time` resistent against the first touch on the screen: if I randomly click somewhere else or scroll, `q-time` will react to every next click; only if clicked first, it ignores it.
- If the first click on `q-time` is turned into a swipe event, it starts reacting to it.

I could easily imagine that this is a broader issue occuring in iOS PWAs and potentially not related to only `q-time`. On the other hand, all other components react right away to any first click, that's why so far I'm focusing on the `q-time` issue.

I will continue to dig deeper and pot. setup a new clean pwa project with just a `q-time` but as I've already heard many other pwa-related bugs, I would love to hear opinions how/where to look for potential debugging/solutions.

**Comment by pdanpdan** [comment]
This might be related to fastClick, but I have to check on all platforms. I'll post updates here.

**Comment by rstoenescu** [comment]
The rendering engine on iOS for PWAs is ancient (that's the least it could be said). It's not the installed Safari, so issues like this may appear.

Thanks for reporting!

**Comment by alexeigs** [comment]
> This might be related to fastClick, but I have to check on all platforms. I'll post updates here.

Is there a temporary workaround to imitate a first click/touch for those views that currently are an issue at time of mounting that may let the first user touch to be recognized from the beginning?

**Comment by alexeigs** [comment]
> This might be related to fastClick, but I have to check on all platforms. I'll post updates here.

@pdanpdan  Is there a temporary workaround to imitate a first click/touch for those views that currently are an issue at time of mounting that may let the first user touch to be recognized from the beginning?

**Comment by pdanpdan** [comment]
Try to emit a touch event (touchstart) on document.body when the application is loaded. See if it helps.

**Comment by alexeigs** [comment]
> Try to emit a touch event (touchstart) on document.body when the application is loaded. See if it helps.

@pdanpdan  The following does the trick, thanks! Hope this also helps you for debugging... Let me know if you believe it's not a bug but an issue in my setup.

```
    mounted() {
      const el = document.getElementById('time-picker')!
      el.click()
    }
```

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.5.2.
